root.spec

```diff
@@ -3,8 +3,8 @@
 ## INITENV +PATH PYTHON3PATH %{i}/lib
 ## INITENV SET ROOTSYS %{i}
 
-%define tag 384e06fed3d7305d8d438c34399d6e6e327256de
-%define branch cms/master/73ae672
+%define tag a2847019732b1b02e0cb1e437b4b9f4c913ddee1
+%define branch cms/master/e2c9e16
```

Comment:
please open a new PR with only this change. All of other changes are needed now ... right?

Reply:
I'll keep the branch if ROOT decide to ditch glew, apart from that yes no need to keep this one open. This is not merged yes so it also needs it https://github.com/root-project/root/pull/5583